src/color/getColorContrastScores.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import getColorContrast from './getColorContrast'
+
+import type { ContrastScores } from '../types/color'
+
+/**
+ * Returns the contrast scores between two colors based on the
```

Comment:
Should update the description since it is not returning scores but instead checking the codes compliance with various guideline values.